integration-tests/typescript-angular/src/generated/api.github.com.yaml/client.service.ts

```diff
@@ -320,8 +320,44 @@ import { HttpClient, HttpParams, HttpResponse } from "@angular/common/http"
 import { Injectable } from "@angular/core"
 import { Observable } from "rxjs"
 
+export type GitHubV3RestApiServiceServer =
+  Server<"GitHubV3RestApiServiceServer">
+
+export class GitHubV3RestApiServiceServers {
+  static default(): GitHubV3RestApiServiceServer {
+    return "https://api.github.com" as GitHubV3RestApiServiceServer
+  }
+
+  static specific(url: "https://api.github.com") {
+    switch (url) {
+      case "https://api.github.com":
+        return {
+          with(): GitHubV3RestApiServiceServer {
+            return "https://api.github.com" as GitHubV3RestApiServiceServer
+          },
+        }
+    }
+  }
+
+  static custom(url: string): GitHubV3RestApiServiceServer {
```

Comment:
This should probably return a different branded type that we can union into the individual operation signatures